docs/source/complaints-award.rst

```diff
@@ -82,12 +82,30 @@ Or declines it:
 .. include:: tutorial/award-complaint-decline.http
    :code:
 
+Correcting problems
```

Comment:
Correcting problems
-------------------

If tender award complaint was satisfied by the Complaint Review Body, then procuring entity has to cancel this award and start the awarding process again.

Cancelling award:

.. include:: tutorial/award-complaint-satisfied-resolving.http
   :code:

After award cancellation system generates new award. Its location is present in the `Location` header of response.

.. include:: tutorial/award-complaint-newaward.http   
   :code:

Submitting Resolution Confirmation
----------------------------------

When complaint was successfully resolved complainant submits resolution confirmation.

.. include:: tutorial/award-complaint-resolved.http
   :code:

Submitting complaint to new award
-----------------------------